### PR TITLE
Starting work on blog links and nx_cugraph docs 

### PR DESCRIPTION
**Pull request by acostadon**
This PR starts to address updating cuGraph docs to reflect recent blogs and work on nx_cugraph as a backend for NetworkX

### CODE REVIEW

**Comment by acostadon** [comment]
rerun tests

**Comment by acostadon** [comment]
rerun tests

**Review by BradReesWork — APPROVED** [review]

File: docs/cugraph/source/basics/nx_cuGraph_algos.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#########################################
+nx_cugraph as a backend for NetworkX code
+#########################################
+
+Wheras previous versions of cuGraph have included mechanisms to make it
```
[the remark this answers is not in the document]
Comment (acostadon) [reply docs/cugraph/source/basics/nx_cuGraph_algos.rst]: fixed. This file is removed and superceded by source/nx_cugraph/nx_gugraph.md

**Pull request merged**